.circleci/config.yml

```diff
@@ -764,8 +777,20 @@ jobs:
     environment:
       <<: *default-build-environment
     steps:
-      - checkout
       - prepare-for-build
+      - run:
+          name: Install go as needed
+          command: |
+            command -v go >/dev/null || (
+              wget https://go.dev/dl/go1.18.3.linux-amd64.tar.gz
+              tar -C /usr/local -xzf go1.18.3.linux-amd64.tar.gz
+              rm go1.18.3.linux-amd64.tar.gz
+
+              echo 'export GOPATH=/usr/local/go' >> $BASH_ENV
+              echo 'export PATH=$PATH:$GOPATH/bin' >> $BASH_ENV
+            )
+            . $BASH_ENV
+            go version
```

Comment:
Is this `go version` a debug thing left over?
I could also see wanting to have the version info in the CI output

Reply:
It was useful for verifying go is installed after that check, as well as printing the version. I can remove it if you prefer.